src/content/styleguide.mdx

```diff
@@ -42,40 +43,71 @@ contact-form-component.stories.tsx
 contact-form-component.test.tsx
 contact-form-component.e2e.tsx
 user-profile.fragment.ts
-user-profile.interface.ts
+user-profile.types.ts
 get-user-profile.request.ts
 parse-user-profile.ts
 ```
 
 ## Folders structure and files organisation
 
-Folders structure of the whole project should be clear and easy to navigate. A project `src` should contain defined structure:
-- `/components` is an obvious directory to store all components. This folder has also its own structure inside:
+Folders structure of the whole project should be built with _domain driven development_ in mind. This will make it clear and easy to navigate. A project `src` directory should contain only domains (modules). Each module should be responsible for a separate functionality, its API should be clear, obvious and what's the most important - it shouldn't cause circular dependency with other modules. If it does, its worth to rethink and refactor the code (maybe split it into even smaller modules?).
```

Comment:
```suggestion
Folders structure of the whole project should be built with _domain driven design_ principles in mind. This will make it clear and easy to navigate. A project's `src` directory should contain only domains modules. Each module should be responsible for a separate functionality, its API should be clear, obvious and—what's the most important—it shouldn't cause circular dependency issue with other modules. If it does, its worth to rethink and refactor the code (maybe split it into even smaller modules?).
```

Just some editorial suggestions. 🟢